modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/DriverInfo/DriverInfoForm.tsx

```diff
@@ -47,42 +57,147 @@ const DriverInfoForm = ({
 
 	const [occupationValue, setOccupationValue] = useState<string>();
 
+	const [options, setOptions] = useState<any[]>([]);
+
+	const [
+		_millitaryAffiliationOptions,
+		setMillitaryAffiliationOptions,
+	] = useState<any[]>([]);
+
+	const [_millitaryAffiliation, setMillitaryAffiliation] = useState<string>(
+		''
+	);
+
+	enum dropdownAlign {
+		topCenter = 0,
+		topRight = 1,
+		middleRight = 2,
+		bottomRight = 3,
+		bottomCenter = 4,
+		bottomLeft = 5,
+		middleLeft = 6,
+		topLeft = 7,
+	}
+
+	type hasRequiredErrorTypes = {
+		accidentCitation: boolean;
+		ageFirstLicenced: boolean;
+		firstName: boolean;
+		gender: boolean;
+		governmentAffiliation: boolean;
+		highestEducation: boolean;
+		lastName: boolean;
+		maritalStatus: boolean;
+		millitaryAffiliation: boolean;
+		occupation: boolean;
+		otherOccupation: boolean;
+		relationToContact: boolean;
+	};
+
+	const hasRequiredError: hasRequiredErrorTypes = {
+		accidentCitation: false,
+		ageFirstLicenced: false,
+		firstName: false,
+		gender: false,
+		governmentAffiliation: false,
+		highestEducation: false,
+		lastName: false,
+		maritalStatus: false,
+		millitaryAffiliation: false,
+		occupation: false,
+		otherOccupation: false,
+		relationToContact: false,
+	};
+
+	const [hasError, setHasError] = useState(hasRequiredError);
+
+	const ErrorMessage = ({text}: any) => (
+		<div className="form-feedback-group">
+			<div className="form-feedback-item">
+				<span className="form-feedback-indicator">
+					<ClayIcon symbol="info-circle"></ClayIcon>
+				</span>
+
+				{text ? text : 'This field is required'}
+			</div>
+		</div>
+	);
+
 	const [accidentCitationValue, setAccidentCitationValue] = useState<boolean>(
 		false
 	);
 
 	const accidentCitations = form.map((form) => {
-		return form.accidentCitation.filter(
-			(accidentCitation: {id: number}) => accidentCitation.id !== id
+		const list = form.accidentCitation || [];
+
+		return list.filter(
+			(accidentCitation: {id: number}) =>
+				accidentCitation.id !== form.accidentCitation.id
```

Comment:
What does "list" mean? Try something like: 
```suggestion
		const listOfAccidentCitations = form.accidentCitation || [];

		return listOfAccidentCitations.filter(
			(accidentCitation: {id: number}) =>
				accidentCitation.id !== form.accidentCitation.id
```